clouddriver-kubernetes/src/main/groovy/com/netflix/spinnaker/clouddriver/kubernetes/api/KubernetesClientApiAdapter.groovy

```diff
@@ -123,6 +124,19 @@ class KubernetesClientApiAdapter {
       }
     }
   }
+  static String getDeploymentRevision(V1beta1StatefulSet statefulSet) {
```

Comment:
I don't think deployment revisions make sense in the context of stateful sets. Those are particular to the relationship between replica sets and deployment objects.